src/Options/index.js

```diff
@@ -64,6 +64,8 @@ export interface ParseServerOptions {
   webhookKey: ?string;
   /* Key for your files */
   fileKey: ?string;
+  /* Cache-Control header for files router */
+  filesCacheControl: ?string;
```

Comment:
Note: definitions are generated from this file, just add the default value after an run the script in resources/buildConfigDefinitions.js